components/tyk-mdcb/templates/deployment-mdcb.yaml

```diff
@@ -78,7 +78,7 @@ spec:
             - name: TYK_MDCB_STORAGE_ADDRS
               value: {{ include "mdcb.redis_url" . | quote }}
             - name: TYK_MDCB_HASHKEYS
-              value: "{{ .Values.mdcb.hashKeys }}"
+              value: {{ .Values.global.hashKeys | default .Values.mdcb.hashKeys | quote }}
```

Comment:
in my opinion, no need to check for `.Values.mdcb.hashKeys`. we can remove it as `tyk-mdcb` chart is still beta; hence, breaking changes can be introduced.

So, for MDCB chart, let's remove `.Values.mdcb.hashKeys` field and only check for `.Values.global.hashKeys` for